SR2019/2018-10-16-sr2019-full.md

```diff
@@ -0,0 +1,11 @@
+---
+to: Student Robotics 2019 Intrest
+subject: The London kickstart is full
+note: This email is to be sent to those who are _not_ in the 'Student Robotics 2019' list.
+---
+
+Hello!
+
+We regret to inform you that Student Robotics 2019 is full. This year we are running on reduced resources, and are unable to host as many teams as we have in previous years.
```

Comment:
Do we want to say we will contact them if spaces become available e.g if teams drop out very early on,

Reply:
Probably not. I think it was decided that it would be better to allow an existing team a second kit rather than invite a new team in, given they'll have a lot less time

Reply:
If we get a dropout before about January I'd be up for contacting one of these teams to invite them in. I  don't think they'd be at much disadvantage and we'd reach more people.
I don't feel particularly strongly about this though.